src/Net/IHttpListenerRequest.cs

```diff
@@ -0,0 +1,333 @@
+﻿using System;
+using System.Collections.Specialized;
+using System.IO;
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Rothko
+{
+    /// <summary>
+    /// Describes an incoming HTTP request to an System.Net.HttpListener object.
+    /// </summary> 
+    public interface IHttpListenerRequest
+    {
+        /// <summary>
+        /// Gets the MIME types accepted by the client.
+        /// </summary>
+        /// <returns>
+        /// A <see cref="string"/> array that contains the type names specified in the request's
+        /// Accept header or null if the client request did not include an Accept header.
+        /// </returns>
+        string[] AcceptTypes { get; }
```

Comment:
For the most part, Rothko tries to follow the original API as closely as possible. But in cases like this where the API returns an array, I switch it to `IEnumerable<T>`. So in this specific case, this should be `IEnumerable<string>`